includes/sanitizers/class-amp-video-sanitizer.php

```diff
@@ -134,6 +135,13 @@ public function sanitize() {
 				$child_nodes[] = $fallback;
 			}
 
+			if ( empty( $new_attributes['width'] ) && empty( $new_attributes['height'] ) ) {
+				$new_attributes[ Attribute::OBJECT_FIT ] = 'contain';
```

Comment:
Is this not redundant given:

https://github.com/ampproject/amp-wp/blob/f56bd67bdc6978872cce2eff08380b1f34988dd3/assets/css/src/amp-default.css#L9-L11

Reply:
```suggestion
```

Reply:
Another reason to not use `object-fit` attribute is it means stylesheets can't override it, as AMP adds it as an inline `style` which we can't override. See https://github.com/ampproject/amp-wp/issues/6428.

Reply:
> Is this not redundant given...

Oh right I forgot about that style rule, thanks for removing it.

That would also be the case in the image sanitizer then, right?

https://github.com/ampproject/amp-wp/blob/ef4653ecdb6b1cebaaf8083f27b8e8be3d071744/includes/sanitizers/class-amp-img-sanitizer.php#L269-L272

Reply:
Yes indeed. I thought that had been removed. It wouldn't be needed anymore due to the `object-fit:contain` which was added to `.amp-wp-unknown-size ` in #6518. See https://github.com/ampproject/amp-wp/pull/6518/files#r684555247.

So `$node->setAttribute( Attribute::OBJECT_FIT, 'contain' ); ` could be removed here as well.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/6576/commits/6178382e08000cca079b7b4c72c74ab7bebcc185.